org.kde.kget.json

```
@@ -0,0 +1,140 @@
+{
+  "id" : "org.kde.kget",
+  "branch" : "master",
+  "runtime" : "org.kde.Platform",
+  "runtime-version" : "5.15-21.08",
+  "runtime-commit" : "e8e3a76d0c26375c55740919779a70e76843cca2305ec01bc3c5d3cbeae2c9b6",
+  "sdk" : "org.kde.Sdk",
+  "sdk-commit" : "5a0b113ebd0a557ee77d73107176ff5f9489da8db422a41f03e2159d43b078a8",
+  "command" : "kget",
+  "modules" : [
+    {
+      "name" : "boost",
+      "buildsystem" : "simple",
+      "sources" : [
+        {
+          "url" : "https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.bz2",
+          "sha256" : "f0397ba6e982c4450f27bf32a2a83292aba035b827a5623a14636ea583318c41",
+          "type" : "archive"
```

```suggestion
          "type" : "archive"
          "url" : "https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.bz2",
          "sha256" : "f0397ba6e982c4450f27bf32a2a83292aba035b827a5623a14636ea583318c41"
```

same below. it is common practice that "type" is first when declaring a source.